src/project.cpp

```diff
@@ -143,6 +143,7 @@ void Project::clearMapCache() {
         Map *map = mapCache->take(mapName);
         if (map) delete map;
```

Comment:
Why not set `map` to `nullptr` right here, instead?  Seems pointless to hookup the slot action.

Reply:
Oh, this is not the `editor` class